lua/lspconfig/sixtyfps.lua

```diff
@@ -0,0 +1,21 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.sixtyfps = {
+    default_config = {
+        cmd = {'sixtyfps-lsp'},
+        filetypes = {'sixtyfps'},
+        root_dir = function(filename) return util.path.dirname(filename) end
```

Comment:
There has to be a better root marker than this

Reply:
I don't think that it has it
It's just a markup language, that can be used in Rust, C++ and NodeJS projects

Reply:
I'm just generally opposed to adding roots that could be invalid, I may change the mechanism such that servers launch without project roots after 0.5.1 is released.